about/serverless-logic-overview.adoc

```diff
@@ -0,0 +1,29 @@
+:_content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="serverless-logic-overview"]
+= OpenShift Serverless Logic overview
+:context: serverless-logic-overview
+
+toc::[]
+
+OpenShift Serverless Logic enables developers to define declarative workflow models that orchestrate event-driven, serverless applications.
+
+You can write the workflow models in YAML or JSON format, which are ideal for developing and deploying serverless applications in cloud or container environments.
+
+To deploy the workflows in your {ocp-product-title}, you can use the OpenShift Serverless Logic Operator.
+
+// Add additional resources if any
+
+The following sections provide an overview of the various OpenShift Serverless Logic concepts.
+
+// modules present in this assembly 
+
+include::modules/serverless-logic-overview-events.adoc[leveloffset=+1]
+include::modules/serverless-logic-overview-callbacks.adoc[leveloffset=+1]
+include::modules/serverless-logic-overview-jq-expressions.adoc[leveloffset=+1]
+include::modules/serverless-logic-overview-error-handling.adoc[leveloffset=+1]
+//include::modules/serverless-logic-overview-configuration-properties.adoc[leveloffset=+1]
```

Comment:
This module is commented out and does not render in the build preview. Are you sure you want to hide this module from the publication?

Reply:
Confirmed over Slack this is intended. Marking this resolved.

Reply:
That is correct! That module is not needed for the current release but it is needed in the future release so I commented out after the feedback from my SME.